<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="USPHXAWNC0038Y6" name="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer" tests="26" failures="1" timestamp="27 Feb 2017 20:09:51 GMT" time="354.474" errors="1">
  <testcase name="clickStartTransferButton" time="62.974" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickPasswordER" time="0.306" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickSubmitER" time="10.816" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickPassword" time="0.356" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickSearchMyFavButton" time="1.005" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="endACCCall" time="1.003" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="setUpER" time="11.935" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickAgentScritping" time="0.434" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickUserNameER" time="0.617" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickQualify" time="0.102" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickEnrollmentqualtransfer" time="1.000" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer">
    <error type="java.lang.AssertionError" message="exception in clickEnrollmentqualtransfer">
      <![CDATA[java.lang.AssertionError: exception in clickEnrollmentqualtransfer
	at org.junit.Assert.fail(Assert.java:93)
	at edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer.clickEnrollmentqualtransfer(tEST_QualTransfer.java:486)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- clickEnrollmentqualtransfer -->
  <testcase name="clickUserName" time="0.320" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickConsultButton" time="1.003" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="markACCCallDone" time="0.998" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="tearDown" time="1.002" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer">
    <failure type="org.openqa.selenium.remote.UnreachableBrowserException" message="Error communicating with the remote browser. It may have died.
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1cd5757287168e54b817830adce9b0158d&#039;, time: &#039;2016-06-30 19:26:09&#039;
System info: host: &#039;USPHXAWNC0038Y6&#039;, ip: &#039;10.97.11.117&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_101&#039;
Driver info: driver.version: RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=45.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 55083543-21ed-4535-b560-da88481e01af">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'USPHXAWNC0038Y6', ip: '10.97.11.117', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_101'
Driver info: driver.version: RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=45.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 55083543-21ed-4535-b560-da88481e01af
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:665)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:701)
	at org.openqa.selenium.remote.RemoteWebDriver.close(RemoteWebDriver.java:521)
	at edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer.tearDown(tEST_QualTransfer.java:622)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:7055 [/127.0.0.1] failed: Connection refused: connect
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:144)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.execute(NewProfileExtensionConnection.java:160)
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:380)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)
	... 27 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
	... 42 more
]]>
    </failure>
  </testcase> <!-- tearDown -->
  <testcase name="clickInstantCallTransfer" time="1.016" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="markERCallDone" time="1.004" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="getCall" time="152.729" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="setUp" time="15.261" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickCaseInfo" time="55.883" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickTransferDropdown" time="1.005" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickSubmit" time="0.243" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickNameSearch" time="17.273" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickFirstRecord" time="14.175" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="endERCall" time="1.006" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
  <testcase name="clickInstantCallConference" time="1.008" classname="edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer"/>
</testsuite> <!-- edu.apollogrp.tests.UCPSmokeTests.tEST_QualTransfer -->
